Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/submissions-list/RefreshableSubmissionsList.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useCallback } from 'react';
+import { isNil } from 'lodash';
+import SubmissionsList, { ISubmissionsListBaseProps } from './SubmissionsList';
+import { useSubmissionsDetails } from '../../../hooks/submissions/use-submissions-details';
+import Button, { ButtonType } from '../../guidelines/buttons/Button';
+
+interface IRefreshableSubmissionsListProps extends ISubmissionsListBaseProps {
+}
+
+const RefreshablSubmissionsList = ({
```

Comment:
I see no reason to have both:
- `RefreshablaSubmissionsList` that refreshes `items`
- `SubmisssionsList` that has `refresh button`

Remove the `refresh button` from `SubmissionsList` and leave it only in `RefreshablaSubmissionsList`